lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3925,6 +3925,9 @@ namespace {
           }
           if (selfIdx) {
             func->setSelfIndex(selfIdx.value());
+            if (func->getASTContext().LangOpts.hasFeature(
```

Comment:
Is it the case that `func->getASTContext()` is the same as `Impl.SwiftContext`? If so, perhaps you should use the latter, to stay consistent with how the rest of this function is checking `LangOpts`?

Reply:
Yup, the same. Updated the code.